src/components/itemList/SearchBar.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import style from "./SearchBar.module.css";
+
+
+function SearchBar({ handleOrder }) {
+  const handleChange = (e) => {
+    handleOrder(e.target.value);
+  };
+  return (
+    <div className={style.utilBox}>
+      <input
+        className={style.searchBar}
+        placeholder="검색할 상품을 입력해주세요"
+      ></input>
```

Comment:
P3:
children값이 없을 경우 아래처럼 하셔도 됩니다~

```suggestion
      <input
        className={style.searchBar}
        placeholder="검색할 상품을 입력해주세요"
      />
```